cylc/uiserver/workflows_mgr.py

```diff
@@ -293,12 +361,15 @@ async def multi_request(
             req_meta = {}
         req_args = {
             w_id: (
-                self.active[w_id]['req_client'],
+                self.workflows[w_id]['req_client'],
                 command,
                 multi_args.get(w_id, args),
                 timeout,
-            ) for w_id in workflows
-            if w_id in self.active
+            )
+            for w_id in workflows
+            if w_id in self.workflows
+            # skip stopped workflows
+            and self.workflows.get(w_id, {}).get('req_client')
```

Comment:
Think the first condition can be skipped
```suggestion
            # skip stopped workflows
            if self.workflows.get(w_id, {}).get('req_client')
```